inc/saiacl.h

```diff
@@ -1660,10 +1678,43 @@ typedef enum _sai_acl_entry_attr_t
      */
     SAI_ACL_ENTRY_ATTR_FIELD_IPV6_NEXT_HEADER,
 
+    /**
+     * @brief Bridge Type (field mask is not needed)
+     *
+     * This is to match the bridge type of the packet.
+     * This would be useful when the ACL table/group is
+     * bound to a switch bind point and the ACL needs to
+     * be applied only on bridges of type 802.ID or only
+     * on the 802.1Q bridge.The value for this match will
+     * the enum values of sai_bridge_type_t.
+     * i.e. SAI_BRIDGE_TYPE_1Q or SAI_BRIDGE_TYPE_1D.
+     *
+     * @type sai_acl_field_data_t sai_int32_t
+     * @flags CREATE_AND_SET
+     * @default disabled
```

Comment:
_@default_ value doesn't match the type (_sai_bridge_type_t_). disable if maps to 0 then from the _sai_bridge_type_t_, it matches _SAI_BRIDGE_TYPE_1Q_. Not sure if we should add some INVALID/NONE type in the enum. Please think about it.

Reply:
Hi @atitjain , 

I followed the existing convention for defining fields the ACL entry. For example the field SAI_ACL_ENTRY_ATTR_FIELD_ACL_IP_TYPE  uses the enum sai_acl_ip_type_t. 
I guess the disabled value represents the boolean in sai_acl_field_data_t which is present in all ACL fields.